conan/tools/apple/xcodebuild.py

```diff
@@ -12,7 +12,16 @@ def __init__(self, conanfile):
 
     @property
     def _verbosity(self):
-        verbosity = self._conanfile.conf.get("tools.apple.xcodebuild:verbosity", default="", check_type=str)
+        verbosity = self._conanfile.conf.get("tools.build.verbosity")
+        if verbosity:
+            if verbosity not in ("Quiet", "Minimal", "Normal", "Detailed", "Diagnostic"):
+                raise ConanException(f"Unknown value '{verbosity}' for 'tools.build.verbosity'")
+            else:
+                verbosity = {"Quiet": "quiet",
```

Comment:
Probably this is not the best for multi-build-system, because if you provide for example "Minimal" it won't do anything. I think it would be necessary to map all possible input values.

Reply:
I see what you mean, I've centered the mapping to only have normal not affect the output